### DISCUSSION TITLE
Looking for TL-WN821N v4 support

### DISCUSSION THREAD

**Discussion by AwSomeSiz** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#49] [Q&A]
I would be extremely grateful if you could add not only the TL-WN821N v6 model to the list of supported devices, but also the TL-WN821N v4.

**Comment by chris1111** [comment]
unfortunately V4 is only supported in linux and windows

**Reply by AwSomeSiz** [reply]
Okay, but tnx so much for reply :)
